packages/react-scripts/config/webpack.config.js

```diff
@@ -66,6 +66,8 @@ const imageInlineSizeLimit = parseInt(
   process.env.IMAGE_INLINE_SIZE_LIMIT || '10000'
 );
 
+const maxWebpackChunkSize = parseInt(process.env.MAX_WEBPACK_CHUNK_SIZE || '0');
```

Comment:
does 0 mean no max?

Reply:
Yes, read jdsumsion's comment and my reply to him.